CHANGELOG.txt

```diff
@@ -1,3 +1,212 @@
+Airflow 2.1.0, 2021-05-11
+-------------------------
+
+New Features
+""""""""""""
+
+- Add PythonVirtualenvDecorator to Taskflow API (#14761)
+- Add Taskgroup decorator (#15034)
+- Create a DAG Calendar View (#15423)
+- Create cross-DAG dependencies view (#13199)
+- Add rest API to query for providers (#13394)
+- Mask passwords and sensitive info in task logs and UI (#15599)
+- Add SubprocessHook for running commands from operators (#13423)
+- Add DAG Timeout in UI page "DAG Details" (#14165)
+- Add WeekDayBranchOperator (#13997)
+- Add JSON linter to DAG Trigger UI (#13551)
+- Add DAG Description Doc to Trigger UI Page (#13365)
+- Add airflow webserver URL into SLA miss email. (#13249)
+- Add readonly REST API endpoints for users (#14735)
+- Add files to generate Airflow's Python SDK (#14739)
+- Add dynamic fields to snowflake connection (#14724)
+- Add readonly REST API endpoint for roles and permissions (#14664)
+- Add new datetime branch operator (#11964)
+- Add Google leveldb hook and operator (#13109) (#14105)
+- Add plugins endpoint to the REST API (#14280)
+- Add worker_pod_pending_timeout support (#15263)
```

Comment:
```suggestion
- Add ``worker_pod_pending_timeout`` support (#15263)
```

Reply:
This will appear https://airflow.apache.org/docs/apache-airflow/stable/changelog.html in it uses RST syntax -- otherwise this will raise a "spelling" error